### DISCUSSION TITLE
List and delete existing API tokens

### DISCUSSION THREAD

**Discussion by georged** [amberelectric/public-api#124] [Q&A] [answered]
Hi good people of Amber,

just dipping my toes into API, got the token generated, and the data flowing...

My generated token is not listed anywhere (don't expect the token itself, just the name) and I don't see how I can deauthorise/delete one (I _assume_ it's valid indefinitely) 

Help, please?
George

**Comment by madpilot** [comment, marked as answer]
You should see a list of tokens and a delete button

![image](https://user-images.githubusercontent.com/19585/197094857-b1223e64-1e7d-42e9-91c5-3ff64d5a9686.png)

But I think something is wrong at the moment. I'll get someone to have a look.

**Reply by madpilot** [reply]
Yeah, there is a bug there. Unfortunately, we are all sprinting for a big release next week, so I don't think we'll be able to sort it out until later next week.

**Reply by madpilot** [reply]
This is fixed now.
